packages/framework/package.json

```diff
@@ -132,25 +132,26 @@
     }
   },
   "devDependencies": {
+    "@apidevtools/json-schema-ref-parser": "11.6.4",
     "@sveltejs/kit": "^1.27.3",
+    "@types/aws-lambda": "^8.10.141",
     "@types/express": "^4.17.13",
     "@types/node": "^20.14.10",
-    "@types/aws-lambda": "^8.10.141",
     "@types/sanitize-html": "2.11.0",
     "@vercel/node": "^2.15.9",
+    "aws-lambda": "^1.0.7",
+    "eslint-plugin-unused-imports": "^4.0.0",
```

Comment:
What if we moved this to the top-level package.json, disabled the rule by default, then just enabled the rule for this package (given that we extend the top-level eslint config). That way other packages can slowly opt-in and we keep a consistent eslint plugin version through the codebase.